src/DatePicker/DateDisplay.js

```diff
@@ -47,7 +47,7 @@ function getStyles(props, context, state) {
       marginBottom: 10,
     },
     yearTitle: {
-      cursor: (!selectedYear && !props.disableYearSelection) ? 'pointer' : 'default',
+      cursor: props.disableYearSelection ? 'not-allowed' : (!selectedYear ? 'pointer' : 'default'),
```

Comment:
From [docs](http://www.material-ui.com/#/components/date-picker):
`disableYearSelection`	-	Disables the year selection in the date picker.

I thinks, this is the use-case.
Could you please explain, why this is not the use-case?